external_search.py

```diff
@@ -383,9 +383,9 @@ def _f(s):
 
         clauses = []
         if languages:
-            clauses.append(dict(terms=dict(language=languages)))
+            clauses.append(dict(terms=dict(language=list(languages))))
```

Comment:
What was languages before? I'm a little concerned that if it comes in a string 'eng' this will turn it into `['e', 'n', 'g']`.

Or did this just not work at all before?